src/sentry/static/sentry/app/views/eventsV2/index.tsx

```diff
@@ -73,55 +70,33 @@ class EventsV2 extends React.Component<Props> {
       return null;
     }
 
-    const {location} = this.props;
-
-    const sampleQueries = SAMPLE_VIEWS.map((view, index) => {
-      const eventView = EventView.fromEventViewv1(view);
-
-      const to = {
-        pathname: location.pathname,
-        query: {
-          ...eventView.generateQueryStringObject(),
-        },
-      };
-
-      return (
-        <BannerButton
-          to={to}
-          icon="icon-circle-add"
-          key={index}
-          onClick={() => {
-            trackAnalyticsEvent({
-              eventKey: 'discover_v2.prebuilt_query_click',
-              eventName: 'Discoverv2: Click a pre-built query',
-              organization_id: this.props.organization.id,
-              query_name: eventView.name,
-            });
-          }}
-        >
-          {view.buttonLabel || eventView.name}
-        </BannerButton>
-      );
-    });
-
     return (
       <Banner
         title={t('Discover')}
-        subtitle={t('Here are a few sample queries to kick things off')}
+        subtitle={t('Customize your query searches')}
         onCloseClick={this.handleClick}
       >
-        {sampleQueries}
+        <Button>Build a new query</Button>
```

Comment:
Would you be able to add a CTA for this button?

Something like

```
// DEFAULT_EVENT_VIEW_V1 is from data.tsx
const eventView = EventView.fromEventViewv1(DEFAULT_EVENT_VIEW_V1);	

const to = {	
    pathname: location.pathname,	
    query: {	
        ...eventView.generateQueryStringObject(),	
    },	
};

<Button
    to={to}	
    onClick={() => {	
        trackAnalyticsEvent({	
            eventKey: 'discover_v2.prebuilt_query_click',	
            eventName: 'Discoverv2: Click a pre-built query',	
            organization_id: this.props.organization.id,	
            query_name: eventView.name,	
        });	
    }}
>...</Button>
```

Reply:
Yep!